### PR TITLE
Add PSMDB 1.12.0 to PMM and contribution guideline.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
...
+```json lines
+{
+    "image_path": "[docker image name]",
+    "image_hash": "[docker image hash]",
+    "status": "recommended", // or "available"
+    "critical": false // or true
+}
+```
+3. image path and image hash can be taken from docker hub or by running `docker inspect`
+4. status can be `recommended` or `available`
```
Comment (tplavcic) [inline README.md:36]: I think only one version can be `recommended` and multiple can be `available` so maybe add this info.

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+
+## How to create a new docker image
+`make docker-push` will create and push docker image with your changes.
```
Comment (tplavcic) [inline README.md:39]: maybe mention the format of the tag which will be used for the pushed image: `<git_branch>-<short_git_hash>`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BupycHuk) [reply README.md:36]: actually no, I've seen that more than one MongoDB version has a `recommended` status for the PSMDB operator.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: README.md (hunk 2 above)
Comment (BupycHuk) [reply README.md:38]: Done

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+
+For each product we have multiple files in `sources` directory.
+Each file contains supported components list for exact version of product.  
+For example `pmm.2.29.0.pmm-server.json` contains supported components for PMM Server 2.29.0.
+
+
+## How to add new product version into supported versions list.
+
+In this example I'll use PMM Server and PSMDB operator.
+PMM Server uses PSMDB Operator.
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply README.md]: Done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: README.md (hunk 1 above)
Comment (tplavcic) [reply README.md:35]: For operators only one can be `recommended` per major database version, so for PSMDB 5.0 only one version can be recommended and another one can be recommended for PSMDB 4.4.

**Review by tplavcic — APPROVED** [review]

**Pull request merged**